dbqueries.py

```diff
@@ -114,11 +122,59 @@ class QueryDict(TypedDict):
             }
         }
     },
+    'udw_duplicate_course_ids': {
+        'output_file_name': 'udw_duplicate_course_ids.csv',
+        'data_source': 'UDW',
+        'query_name': 'UDW Duplicate Course IDs',
+        'type': 'standard',
+        'query': '''
+            select c.canvas_id, count(c.canvas_id)
+            from course_dim c
+            left join enrollment_term_dim t
+                on t.id=c.enrollment_term_id
+            where c.workflow_state != 'deleted'
+                and t.date_start <= current_timestamp
+                and t.date_start + interval '120 days' >= current_timestamp
+            group by c.canvas_id
+            having count(*) > 1;
+        ''',
+        'checks': {
+            'less_than_two': {
+                'color': 'RED',
+                'condition': LESS_THAN_TWO,
+                'rows_to_ignore': []
+            }
+        }
+    },
```

Comment:
This approach works, but I'm shoehorning it into the existing approach a bit because really there won't be any rows to check if there aren't any grouped records that satisfy `having count(*) > 1`. We could provide some alternative checking approach that checks for the existence of rows in the first place. IDK, maybe I'm overthinking it.